src/dashboard-below-map-view.js

```diff
@@ -71,6 +71,11 @@ module.exports = cdb.core.View.extend({
   },
 
   _onWidgetsChange: function () {
+    this._widgets.each(function (widget) {
+      if (widget.get('type') === 'time-series') {
```

Comment:
I'd use a constant here, since we're using that string multiple times in the file

Reply:
Yep